include/static-pages.php

```diff
@@ -122,23 +124,25 @@ public function page_link( $link, $id ) {
 	}
 
 	/**
-	 * Adds page_on_front and page_for_posts properties to the language objects.
+	 * Adds page_on_front and page_for_posts properties to language data before the object is created.
 	 *
-	 * @since 1.8
+	 * @since 3.4
 	 *
-	 * @param PLL_Language[] $languages The list of languages.
-	 * @param PLL_Model      $model     The instance of PLL_Model.
-	 * @return PLL_Language[]
+	 * @param array $static_pages Array of language page_on_front and page_for_posts properties.
+	 * @param array $language Language data.
+	 * @return array Language data with static pages ids.
```

Comment:
```suggestion
	 * @return array Static pages properties if exist.
```

Reply:
Done in 2db4770fd0222874731ea19e18cfac4184d91794